components/browser/state/src/main/java/mozilla/components/browser/state/reducer/TabListReducer.kt

```diff
@@ -62,7 +63,16 @@ internal object TabListReducer {
             }
 
             is TabListAction.SelectTabAction -> {
-                state.copy(selectedTabId = action.tabId)
+
+                val updatedTabsList = state.tabs.map {
+                    if (it.id == action.tabId) {
+                        it.lastAccess = System.currentTimeMillis()
+                        Log.log(Log.Priority.DEBUG, "")
```

Comment:
nit: a leftover of debugging :)

Reply:
Instead of changing the value `lastAccess` of directly it will be better to use `copy` to change value

Reply:
+1